portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,6 +62,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable(fetchOptions)) {
       String email = (String) entity.getProperty("email");
       String text = (String) entity.getProperty("text");
+      float score = calculateSentimentScore(text);
```

Comment:
I think it was asked to store the score instead of calculating the score when you need it.